backend/src/entities/UserEntity copy.ts

```diff
@@ -0,0 +1,8 @@
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+import { UserEntity } from './UserEntity';
+
+@Entity()
+export class AdminEntity extends UserEntity {
```

Comment:
What is this ??????????????????????????????????????????????????

Reply:
Oupsies